src/extensions/default/JavaScriptCodeHints/main.js

```diff
@@ -341,10 +341,11 @@ define(function (require, exports, module) {
     };
 
     /**
-     * @return {boolean} - true if the document is a html file
+     * @return {boolean} - true if the document is a html or php file
      */
     function isHTMLFile(document) {
```

Comment:
Function name is now misleading. maybe change including usages to
isHTMLOrPHPFile(document)
or alternatively move the check to a more localized place.